pandas/tests/frame/methods/test_select_dtypes.py

```diff
@@ -128,6 +128,36 @@ def test_select_dtypes_exclude_include_using_list_like(self):
         e = df[["b", "e"]]
         tm.assert_frame_equal(r, e)
 
+    def test_select_dtypes_exclude_include_int(self):
+        # Fix select_dtypes(include='int') for Windows, FYI #36596
+        df = DataFrame(
+            {
+                "a": list("abc"),
+                "b": list(range(1, 4)),
+                "c": np.arange(3, 6, dtype="int32"),
+                "d": np.arange(4.0, 7.0, dtype="float64"),
+                "e": [True, False, True],
+                "f": pd.date_range("now", periods=3).values,
+            }
+        )
+        exclude = (np.datetime64,)
```

Comment:
can you parameterize on these types (e.g. you can add include & excude as paramters), the this is much simpler to grok.

Reply:
Yes, that will be nicer for sure. I updated this pull request with the requested changes.